test/integration/patients/worklist.js

```diff
@@ -32,7 +34,7 @@ context('worklist page', function() {
             details: null,
             duration: 0,
             due_date: null,
-            updated_at: moment().format(),
+            updated_at: moment.utc().format('YYYY-MM-DD'),
```

Comment:
no format arg here :-D

Reply:
ok so...why this comment from earlier? 
https://github.com/RoundingWell/care-ops-frontend/pull/172#discussion_r358941041

i think this is where some of my confusion is coming from...

Reply:
oh whoops, I keep looking at different parts of this test